src/ceph-volume/ceph_volume/util/prepare.py

```diff
@@ -83,7 +83,12 @@ def get_block_db_size(lv_format=True):
     .. note: Configuration values are in bytes, unlike journals which
              are defined in gigabytes
     """
-    conf_db_size = conf.ceph.get_safe('osd', 'bluestore_block_db_size', None)
+    conf_db_size = None
+    try:
+        conf_db_size = conf.ceph.get_safe('osd', 'bluestore_block_db_size', None)
+    except RuntimeError:
+        logger.debug("failed to load ceph configuration, will use defaults")
```

Comment:
add a `logger.exception` here so that if something else not related to the missing conf.ceph can be seen in the logs